### PR TITLE
Fixes proc reading when in an included file

### PR DESCRIPTION
**Pull request by jonnyh404**

### CODE REVIEW

**Review by cflynn3 — APPROVED** [review]
Looks good to me.

File: ResSimpy/Nexus/NexusNetwork.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def __load_procs(self) -> list[NexusProc]:
         for file in files_dict.values():
 
             # for every surface file, grab the lines as a list
-            file_contents = file.file_content_as_list
+            file_contents = file.get_flat_list_str_file
```
Comment (kwr3) [inline ResSimpy/Nexus/NexusNetwork.py:121]: nice fix here in the for loop

File: tests/Nexus/nexus_simulator/test_nexus_constraints.py [hunk 2]
```diff
@@ -400,14 +400,22 @@ def test_nexus_constraints_skip_procs(mocker):
     # Arrange
     surface_content = '''TIME 04/10/2019
 
-PROCS NAME PROCNAME 
+PROCS NAME START_SEAM_TUNING  CLEAR PRIORITY 2
```
Comment (kwr3) [inline tests/Nexus/nexus_simulator/test_nexus_constraints.py:403]: good set of tests

**Review by kwr3 — APPROVED** [review]
looks good!

**Pull request merged**